netctl/commands.go

```diff
@@ -88,22 +88,22 @@ var Commands = []cli.Command{
 				ArgsUsage: "[network]",
 				Flags: []cli.Flag{
 					tenantFlag,
-					cli.BoolFlag{
-						Name:  "public, p",
-						Usage: "Indicate this network is public",
-					},
 					cli.StringFlag{
 						Name:  "encap, e",
-						Usage: "Packet tag",
+						Usage: "Packet encap",
 						Value: "vxlan",
 					},
 					cli.StringFlag{
+						Name:  "pktTag, p",
```

Comment:
can we keep consistent case with our `--` arguments? this is the only one that has a capital letter in it.